### DISCUSSION TITLE
tquic是否可以跟mqtt实现mqtt over quic

### DISCUSSION THREAD

**Discussion by wanyt** [Tencent/tquic#249] [Q&A (问题和回答)] [answered]
如标题，帮忙解答，多谢。

**Comment by glime-ai** [comment]
您好！请提供具体的问题或者需要解答的内容，以便我能更准确地为您提供帮助。谢谢！

This comment was generated by [Glime](https://glimelab.ai/?utm_source=44520fce49ba43798485a628d6928e22&utm_medium=github).

**Comment by iyangsj** [comment, marked as answer]
MQTT基于TCP协议，目前暂无MQTT over QUIC的协议标准。
你可以按需定制将MQTT实现与tquic集成，实现MQTT over QUIC
